lib/udf_string_utils.rb

```diff
@@ -244,6 +244,55 @@ class UdfStringUtils
                            {query: "select ?(1, 3, 1, ',')", expect: "1,2" , example: true},
                            {query: "select ?(1, 5, 2, '|')", expect: "1|3", example: true},
                        ]
+      }, {
+          type:        :function,
```

Comment:
@CanCeylan I have checked your code and it looks perfectly fine. Let me know if you are still having the issue I'll pull the branch and debug it more thoroughly.